packages/contracts/package.json

```diff
@@ -37,18 +37,19 @@
     "url": "https://github.com/aragon/osx/issues"
   },
   "homepage": "https://github.com/aragon/osx#readme",
+  "dependencies": {
+    "@ensdomains/ens-contracts": "0.0.11",
+    "@openzeppelin/contracts": "4.9.5",
+    "@openzeppelin/contracts-upgradeable": "4.9.5"
+  },
   "devDependencies": {
     "@aragon/osx-ethers-v1.2.0": "npm:@aragon/osx-ethers@1.2.0",
     "@aragon/osx-v1.0.1": "npm:@aragon/osx@1.0.1",
-    "@aragon/osx-v1.3.0-rc0.2": "npm:@aragon/osx@1.3.0-rc0.2",
+    "@aragon/osx-v1.3.0": "npm:@aragon/osx@1.3.0",
     "@defi-wonderland/smock": "^2.3.4",
-    "@ensdomains/ens-contracts": "0.0.11",
     "@nomicfoundation/hardhat-chai-matchers": "^1.0.5",
     "@nomicfoundation/hardhat-verify": "^1.0.4",
     "@nomiclabs/hardhat-ethers": "^2.2.1",
-    "@opengsn/contracts": "2.2.6",
```

Comment:
OpenGSN was used for meta transactions. However, we don't actively support meta txns momentarily and don't use the custom ERC2771 `Context` or trusted forwarder implementations from `opengsn/contracts` anymore.